pkg/fsm/metrics/sink.go

```diff
@@ -57,6 +59,16 @@ func NewSink() *Sink {
 			},
 			suspendGaugeLabel{}.names(),
 		),
+		processingDurationHistogram: prometheus.NewHistogramVec(
+			prometheus.HistogramOpts{
+				Name: "achilles_processing_duration_seconds",
+				// stay consistent with controller-runtime's reconciliation duration metric, https://github.com/kubernetes-sigs/controller-runtime/blob/9516c0f9a0aa83a499b5a25907899e4edb0dd9db/pkg/internal/controller/metrics/metrics.go#L61-L62
+				Buckets: []float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0,
+					1.25, 1.5, 1.75, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30, 40, 50, 60},
+				Help: "Histogram of the time taken to process an object spec update",
```

Comment:
can we define process a bit more explicitly here? this may help people less familiar with the metric:
```suggestion
				Help: "Histogram of the time taken to process an object spec update from the k8s watch event in the controller to the final FSM state being reached",
```

Reply:
I wordsmithed it a little bit